.changeset/loud-gifts-hang.md

```diff
@@ -0,0 +1,5 @@
+---
+'sv': patch
+---
+
+chore: update internal project structure
```

Comment:
We merged a bunch of prs without a changeset that were just internal. I think we should do the same here.